kupogo.go

```diff
@@ -20,6 +20,8 @@ import (
 	"fmt"
 	"io"
 	"net/http"
+
+	"github.com/go-playground/validator"
```

Comment:
According to the README for this project, you probably want `github.com/go-playground/validator/v10`. You've got a much older version